Source/Diagnostics/WarpXIO.cpp

```diff
@@ -616,7 +616,17 @@ WarpX::WritePlotFile () const
         }
     }
 
-    mypc->WritePlotFile(plotfilename);
+#ifdef WARPX_USE_OPENPMD
+    // Write openPMD format: only for level 0
+    Timer t("openPMD plot");
```

Comment:
are the leftover timers here from development? ideally, let us just integrate all the I/O into the AMReX [tinyprofiler](https://amrex-codes.github.io/amrex/docs_html/AMReX_Profiling_Tools_Chapter.html). Verbose printing to `stdout` is probably not ideal for users.

Reply:
OK